lib/style.css

```diff
@@ -461,3 +461,25 @@ button.goog-buttonset-default:hover {
 .material-detail-view .top-tracks {
   background-color: <<BACK_PRIMARY>>;
 }
+
+paper-button#unsubscribe-playlist-button .playlist-subscribed,
+paper-button#unsubscribe-playlist-button:hover .playlist-unsubscribe {
+    color: <<FORE_PRIMARY>>;
+}
+
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:unsubscribe"] path:last-child,
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:subscribed"] path
+{
```

Comment:
We should prob keep the opening brace on the same line as the last selector for consistency in our CSS